packages/api/src/routes/medical/patient.ts

```diff
@@ -207,6 +208,7 @@ router.delete(
       facilityId,
     };
     await deletePatient(patientDeleteCmd);
+    await deleteAllPatientMappings({ cxId, patientId: id });
```

Comment:
Can we have this on the command instead of the route, please?

Also, it can be executed in parallel w/ the removal of the patient from the DB, right?

Reply:
Moved to command and put in existing Promise.all.